resources/monitoring/templates/grafana/configmaps-datasources.yaml

```diff
@@ -32,7 +32,10 @@ data:
 {{- end }}
 {{- end }}
 {{- end }}
```

Comment:
With that you destroy the idea of having additionalDatasources configurable in a flexible way. I suggest to just add full new entries and keep the additionalDataSources for the customer as it is:
```
{{- if .Values.grafana.dataSources.loki.enabled }}
{{ toYaml .Values.grafana.dataSources.loki.config | indent 4}}
{{- end }}
{{- if .Values.grafana.dataSources.jaeger.enabled }}
{{ toYaml .Values.grafana.dataSources.jaeger.config | indent 4}}
{{- end }}
{{- if .Values.grafana.additionalDataSources }}
{{ toYaml .Values.grafana.additionalDataSources | indent 4}}
{{- end }}